content/content.js

```diff
@@ -66,29 +55,3 @@ function submitForm(formUrl, submitNumber) {
 
   form.submit();
 }
-
```

Comment:
Thanks for cleaning all these unused codes haha. Somehow I feel like there was a reason why I didn't clean this up

Reply:
Maybe your thought process at that time, you still want to support the feature locally instead of going through the server?

Reply:
Can't recall haha